ntc_templates/templates/cisco_xr_show_running-config_hostname.textfsm

```diff
@@ -2,7 +2,7 @@ Value HOSTNAME (\S+)
 
 Start
   # Capture date output by XR by 'show run hostname' first
-  ^\S+ \S+ \S+ .*$$
+  ^\S+\s+\S+\s+\d+\s+\d+:\d+:\d+\.\d+\s+\S+$$
   # Capture hostname
   ^hostname ${HOSTNAME}$$
```

Comment:
Let's take care of another literal space and replace it with a regex too.
Minor change that I'll send in.

```suggestion
  ^hostname\s+${HOSTNAME}$$
```